### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by XiaoHao-Chen** [xuanqing94/AdvGAN#3]
Thank you very much for your excellent work. When I ran the code you provided, I encountered some problems, several of which were not solved. I'd like to consult you.
1: What are the specific meanings of "positive" and "negative" in train.py?
2: In the paper, D_Loss = Ls + Lc1, G_Loss = Lc2 - Ls; but in the program, what I see is
Los_g = λL1 + (1-λ) L2, loss_d =λL1 + (1-λ) L2. Is the definition of loss function in the paper consistent with that in the program?
3: When I validate your program with other data sets, at the step of saving real images, the last four pictures are always black,such as 
![image](https://user-images.githubusercontent.com/46865749/63644819-f3d33a00-c723-11e9-8c05-26c1a6d11857.png). I can't find the reason. I hope you can give me some suggestions.
Thank you again for your help!

**Comment by aknirala** [comment]
Loss in the code does seem to be same fro both gen and dis!!
